dev_guide/expanding_persistent_volumes.adoc

```diff
@@ -0,0 +1,90 @@
+[[expanding_persistent_volumes]]
+= Expanding Persistent Volumes
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+
+toc::[]
+
+[[enabling_expansion_pvc]]
+== Enabling Expansion of Persistent Volume Claims
+
+Volume expansion is a Technology Preview feature, and hence, is not enabled by
+default in your {product-title} 3.9 cluster. There may be other reasons that
+{product-title} administrators wish to enable this feature for certain use
+cases.
+
+[NOTE]
+====
+For more information on Red Hat Technology Preview features support scope,  see
+https://access.redhat.com/support/offerings/techpreview/. 
+====
+
+To allow expansion of persistent volume claims (PVC) by {product-title} users,
+{product-title} administrators must create or update a StorageClass with
+`allowVolumeExpansion` set to `true`. Only PVCs created from that class are
+allowed to expand.
+
+Apart from that, {product-title} administrators must enable the
+`ExpandPersistentVolumes` feature flag and turn on the
+`PersistentVolumeClaimResize` admission controller. Please refer to <link_here>
```

Comment:
We need to replace `<link_here>` with actual link that shows how a admin can enable a feature flag or turn on a admission controller.

Reply:
Can you provide the link? For example:

https://docs.openshift.com/container-platform/3.7/dev_guide/persistent_volumes.html